src/compiler/third_party/tiny-sha1/README.toitware

```diff
@@ -0,0 +1,12 @@
+Name: TinySHA1
```

Comment:
Maybe this file should be README.toit or README.toitlang?

Reply:
I changed it to README.toit.txt.
done.